docs/api/modal.md

```diff
@@ -42,21 +42,23 @@ A Modal is a dialog that appears on top of the app's content, and must be dismis
 
 When using `ion-modal` with Angular, React, or Vue, the component you pass in will be destroyed when the modal is dismissed. As this functionality is provided by the JavaScript framework, using `ion-modal` without a JavaScript framework will not destroy the component you passed in. If this is a needed functionality, we recommend using the `modalController` instead.
 
-TODO: Playground Example
-
 #### Using triggers
 
 A trigger is an element reference, that when selected will automatically open a modal. Triggers are useful when presenting a modal from a button or similar interaction.
 
-TODO: Playground Example
+import PresentTriggerExample from '@site/static/usage/modal/presenting/inline/trigger/index.md';
+
+<PresentTriggerExample />
 
 #### Using `isOpen`
 
 The `isOpen` property on `ion-modal` allows developers to control the presentation state of the modal from their application state. This means when `isOpen` is set to `true` the modal will be presented and when `isOpen` is set to `false` the modal will be dismissed.
 
 `isOpen` uses one-way data binding, meaning it will not automatically be set to `false` when the modal is dismissed. To keep the state of `isOpen` in sync with your UI state, listen for either `ionModalDidDismiss` or `didDismiss` and set the variable to `false`.
 
-TODO: Playground Example
+import PresentIsOpenExample from '@site/static/usage/modal/presenting/inline/is-open/index.md'; 
+
+<PresentIsOpenExample />
```

Comment:
Rather than just hardcode `isOpen` to `true`, could we have a button that sets it instead? That would be closer to the code people will actually write. It'd also be more clear what's actually happening in the playground, since right now it's hard to tell that you're looking at a modal unless you reset the playground and see the animation.